### PR TITLE
ENCD-5894-restrict-strand-specificity-requirement-to-RNA-libraries

### PR DESCRIPTION
**Pull request by iyoungworth**
updated-fixtures-and-inserts

adjusted-expected-metadata

corrected-other-expected-metadata

### CODE REVIEW

File: src/encoded/schemas/library.json [hunk 1]
```diff
@@ -67,6 +67,33 @@
                 }
             ]
         },
+        "nucleic_acid_term_name": {
+            "oneOf": [
+                {
+                    "required": ["strand_specificity"],
+                        "properties": {
```
Comment (gabdank) [inline src/encoded/schemas/library.json]: I think the indentation here is off?

**Review by gabdank — CHANGES_REQUESTED** [review]
indentation is off

File: src/encoded/schemas/library.json (hunk 1 above)
Comment (gabdank) [reply src/encoded/schemas/library.json]: properties should be same indentation as required

Comment (iyoungworth) [reply src/encoded/schemas/library.json]: Ah you're right- the section below has an extra indent also (L87-92), fixed both

**Review by gabdank — APPROVED** [review]

**Pull request merged**